osu.Game.Tests/NonVisual/FormatUtilsTest.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using NUnit.Framework;
+using osu.Game.Utils;
+
+namespace osu.Game.Tests.NonVisual
+{
+    [TestFixture]
+    public class FormatUtilsTest
+    {
+        [TestCase(0, "0.00%")]
+        [TestCase(0.01, "1.00%")]
+        [TestCase(0.9899, "98.99%")]
+        [TestCase(0.989999, "99.00%")]
```

Comment:
I'm not sure I agree with this test case. I understand that 100% is a special accuracy, but you could argue that if 99.999% doesn't get rounded up to a full percent, then neither should 98.999%, 89.999%, and so forth. I would probably do something like this instead:

```csharp
            int truncated = (int)(accuracy * 10000);
            return $"{truncated / 10000.0f:0.00%}";
```

It's a bit harsh maybe to just truncate, but in a rhythm game I'd say it's warranted to be stricter, and it's a third decimal place anyway. Of course it gets hazy around the round numbers, but again, I'm not sure that 89 to 90% should differ much from 99 to 100%.

Reply:
From the angle that we have other cutoffs for grades which are always round ints, i think you may be making some sense here. Else we were get similar complaints "I got 90% but not an A" or the likes.

Reply:
Have applied this change, I think it feels better.